Appendices/NodeStateDiagrams.rst

```diff
@@ -76,7 +69,6 @@ Assignment nodes transition to ITERATION_ENDED when the ExitCondition
 becomes true; the PostCondition determines the outcome.
 
 .. figure:: ../_static/images/Executing_Assignment_nodes_with_Exit_condition.png

```

Comment:
Per our conversation, this link needs to be updated to the new diagram you created.

Reply:
Done